internal/pkg/model/state.go

```diff
@@ -390,15 +389,11 @@ func (s *State) GetOrCreate(key Key) (ObjectState, error) {
 	} else {
 		// Create
 		var object ObjectState
-		switch k := key.(type) {
+		switch key.(type) {
 		case BranchKey:
 			object = &BranchState{}
 		case ConfigKey:
-			if component, err := s.components.Get(*k.ComponentKey()); err == nil {
-				object = &ConfigState{Component: component}
-			} else {
-				return nil, err
-			}
+			object = &ConfigState{}
```

Comment:
Pri vytvarani `ConfigState` sa nemusi nacitavat `Component` + je to rovnako ako pri ostatnych `*State`.